ironfish/src/network/peerNetwork.ts

```diff
@@ -1058,6 +1060,9 @@ export class PeerNetwork {
     const headers = []
     let skipCounter = skip
 
+    // Limit the total number of lookups to avoid excessive disk usage
+    let remainingLookups = MAX_REQUESTED_HEADERS * 2
```

Comment:
Not sure if it made sense to make this a const in version.ts since its pretty specific to this logic right now, open to suggestions on this

Reply:
I think I'd put in in `version.ts` as well, it might make it a bit more clear that adjusting `MAX_REQUESTED_HEADERS` will affect the max lookups if the two numbers are adjacent in `version.ts`